src/pages/components/YoutubeModal.tsx

```diff
@@ -0,0 +1,42 @@
+import { useRef, useState } from 'react'
+import { InfoCircleOutlined } from '@ant-design/icons'
+import { Modal } from 'antd'
+import './YotubeModal.scss'
+
+type InfoYoutubeModalProps = {
+  label: string
+  videoUrl: string
+  title: string
+}
+
+const InfoYoutubeModal = ({ videoUrl, label, title }: InfoYoutubeModalProps) => {
+  const [visible, setVisible] = useState(false)
+  const iframeRef = useRef<HTMLIFrameElement | null>(null)
+
+  function closeAndStop() {
+    const iframe = iframeRef.current
+
+    if (iframe) {
+      iframe.src = iframe.src
+    }
+
+    setVisible(false)
+  }
```

Comment:
let's remove the iframe from the DOM instead of stopping the video by cleaning the src param